misc/docker/README.md

```diff
@@ -27,7 +27,7 @@ store it in the misc/docker directory
  cp {logging,config}.yaml /etc/ansible-runner-service/.
  cp samples/* /usr/share/ansible-runner-service/
```

Comment:
I think better to create this folders, or another possibility (even better) is to provide the folders already created in the tar.gz
mkdir /usr/share/ansible-runner-service/artifacts
mkdir /usr/share/ansible-runner-service/env
mkdir /usr/share/ansible-runner-service/inventory

Reply:
+1. This is something I've asked Daniel to look at once we merge this PR